customer_outstanding_statement/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================================
+Print Customer Outstanding Statement
+====================================
+
+The outstanding statement provides details of all outstanding on the customer receivables
```

Comment:
The outstanding statement provides details of all outstanding customer receivables entries
up to a particular date